lld/test/ELF/aarch64-build-attributes-be.s

```diff
@@ -0,0 +1,50 @@
+// REQUIRES: aarch64
+// RUN: llvm-mc -triple=aarch64_be %s -filetype=obj -o %t.o
+// RUN: ld.lld %t.o --shared -o %t.so
+// RUN: llvm-readelf -n %t.so | FileCheck %s --check-prefix=NOTE
+
+/// The Build attributes section appearing in the output of
```

Comment:
I recommend moving that to just before `NOTE`

For the comment. I think the purpose of the test is to check lld can handle both little and big endian encoding? Perhaps something like

/// Test that lld can read both little and big-endian build-attributes.